src/superRare.ts

```diff
@@ -0,0 +1,16 @@
+import { Transfer } from "../generated/templates/NftContract/ERC721";
+import { NftContract } from "../generated/schema";
+import { fetchName, fetchSymbol, handleTransfer } from "./mapping";
+
+export function handleTransferSuperRare(event: Transfer): void {
+  handleTransfer(event);
+
```

Comment:
@Tibike6, could you please map `creatorAddress` from `creatorOfToken(tokenId)` function of SuperRare contract. You can do it in the same way as I did with Nifty Gateway `nameOfCreator()`

Reply:
Hm, do you know what is the difference between https://etherscan.io/address/0xb932a70a57673d89f4acffbe830e8ed7f75fb9e0 and https://etherscan.io/address/0x41a322b28d0ff354040e2cbc676f0320d8c8850d ?

Reply:
I checked both contracts. There was an update. The first 4434 NFT's are in the first contract and the other 13906 NFT's in V2. I was already a bit surprised that there are so few of them. 😊 I will also integrate the second contract

Reply:
@frostiq 
- added SuperRareV2
- created getAddressOfCreator because the contract function names for getting the creator address has a different name (`creatorOfToken `in V1, `tokenCreator`in V2)

All SuperRare NFT's (currently 18340) are created. You can test the subgraph here: 
https://thegraph.com/explorer/subgraph/tibike6/cryptomedia-test

**What I've noticed:**
Since SuperRare has two contract addresses, you can't query alle NFT's from SuperRare because the id is the contract address and not the platform name. I would suggest to use platform name as id and save an array of contract addresses or we must wait until thegraph supports `OR` filters.

Example query: (0x41a322b28d0ff354040e2cbc676f0320d8c8850d is the contract address of SuperRareV1)
```
  nfts(where: {contract: "0x41a322b28d0ff354040e2cbc676f0320d8c8850d"}, orderBy: createdAt, orderDirection: desc) {
    id
    contract {
      name
    }
  }
```

Reply:
Thanks @Tibike6!
That's a good observation.
But it's still possible to filter by platform name, right?